new_dawn_server/users/api/resources.py

```diff
@@ -284,6 +284,33 @@ def get_and_save_city_pref(bundle):
                 location_list.append(city_pref)
         return location_list
 
+    @staticmethod
+    def get_and_save_question(bundle):
+
+        if not Question.objects.filter(question=bundle['question']).exists():
```

Comment:
Let's add a bool field in data model to reflect if the question is user-defined. And make that true when question is created here.